src/lib/core/util.c

```diff
@@ -191,7 +192,7 @@ fdprintf(int fd, const char *format, ...)
 		res = write(fd, start, len);
 		if (res > 0)
 			total += res;
-		if (res != len)
+		if (res < 0 || (size_t)res != len)
```

Comment:
This function isn't used. You're fixing dead code here...

`total` is `ssize_t` so this function is screwed if `strlen` turns out to be that big.

I don't understand how this could possibly lead to a (security) bug. If arguments are insanely big, we'll write garbage to the file - so what?

Please drop this patch and mark the SVACE warning as "Won't fix" or "False positive".

Reply:
> This function isn't used. You're fixing dead code here...

Yep, missed that, thanks!
Let's remove it altogether then.